Extension/c_cpp_properties.schema.json

```diff
@@ -142,6 +142,16 @@
                             }
                         },
                         "additionalProperties": false
+                    },
+                    "customConfigurationVariables": {
+                        "type": "object",
+                        "description": "Custom properties that can be queried through the command ${cpptools:activeConfigCustomVariable}.",
```

Comment:
Sorry, forgot to comment on this: let's change description to the following to be consistent with name `customConfigurationVariables`

> Custom variables that can be queried through the command ${cpptools:activeConfigCustomVariable} to use for the input variables in  launch.json.